cmd/root.go

```diff
@@ -39,54 +36,3 @@ func Execute() {
 		os.Exit(-1)
 	}
 }
```

Comment:
This should have stayed here? This was just moved here.

Reply:
Fixed.